drivers/bridge/bridge.go

```diff
@@ -490,6 +490,12 @@ func parseNetworkOptions(id string, option options.Generic) (*networkConfigurati
 		config.EnableIPv6 = val.(bool)
 	}
 
+	if val, ok := option[netlabel.Internal]; ok {
+		if internal, ok := val.(bool); ok && internal {
+			return nil, &driverapi.ErrNotImplemented{}
```

Comment:
@chenchun are you planning on pushing the fix to remove this error and handling the bridge case appropriately ?
If am not wrong, this should be fairly simple change in bridge driver. I think this flag could just translate to disabling ip-masquarding, which we already have a flag to disable. Please let us know asap.